htdocs/eventorganization/conferenceorboothattendee_card.php

```diff
@@ -389,7 +389,15 @@
 
 	// Show message
 	$message = '<a target="_blank" rel="noopener noreferrer" href="'.$urlwithroot.'/public/agenda/agendaexport.php?format=ical'.($conf->entity > 1 ? "&entity=".$conf->entity : "");
-	$message .= '&exportkey='.($conf->global->MAIN_AGENDA_XCAL_EXPORTKEY ? urlencode(getDolGlobalString('MAIN_AGENDA_XCAL_EXPORTKEY')) : '...');
+	if (isset($conf->global->MAIN_AGENDA_XCAL_EXPORTKEY)) {
```

Comment:
`$message .= '&exportkey='.urlencode(getDolGlobalString('MAIN_AGENDA_XCAL_EXPORTKEY', '...'));`

Reply:
@frederic34  okay, the PHP warning message does not come back - but also there is still no form to add an attendee?

Reply:
https://github.com/Dolibarr/dolibarr/pull/33884

Reply:
@JonBendtsen is it ok ?

Reply:
@frederic34  I get this form so I think it works

![image](https://github.com/user-attachments/assets/2e47f6e8-19a3-4da1-8b4f-ca35ca852fd3)